Quick and dirty fix for a fatal issue with 'unsigned short' and 'boolean'
Not submitted upstream yet, this probably requires more insight.

--- common/typeutil.tcl~	2015-11-09 19:28:31.623593000 +0100
+++ common/typeutil.tcl	2015-11-10 12:21:11.547774000 +0100
@@ -77,7 +77,7 @@
 	    puts stderr member
 	    return [oprs_type_nva [$type type] $param $string]
 	}
-	string - array - sequence - "unsigned long" - long - float - double - short - int - char - enum {
+      string - array - sequence - "unsigned long" - long - float - double - {*short} - int - char - enum - boolean {
 	 	    return 1
 	}
 	struct {
